compute/resource_job_test.go

```diff
@@ -452,3 +385,157 @@ func TestResourceJobDelete_Error(t *testing.T) {
 	qa.AssertErrorStartsWith(t, err, "Internal error happened")
 	assert.Equal(t, "789", d.Id())
 }
+
+func TestResourceJobCreateSingleNode(t *testing.T) {
+	cluster := Cluster{
+		NumWorkers: 0, SparkVersion: "7.3.x-scala2.12", NodeTypeID: "Standard_DS3_v2",
```

Comment:
please run `make fmt`, i think this is incorrect formatting.

Reply:
I did run it...